tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -173,7 +174,7 @@ public struct TwoByteStr
 public class TestClass
 {
     public string TestString {get; set;}
-	
+
     public TestClass(int number)
```

Comment:
It would be good to add a test for a virtual method on the base class that isn't overridden.